<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportByDebtor</class>
 <widget class="QWidget" name="ReportByDebtor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1285</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="leftMargin">
      <number>45</number>
     </property>
     <property name="rightMargin">
      <number>45</number>
     </property>
     <item>
      <widget class="QPushButton" name="BackButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>52</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>НАЗАД</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="EcoTaxi">
         <property name="text">
          <string>ОТЧЁТ</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="Header">
         <property name="text">
          <string>По Должнику</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="SettingsButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>52</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>НАСТРОЙКИ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>800</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <property name="spacing">
         <number>30</number>
        </property>
        <property name="leftMargin">
         <number>50</number>
        </property>
        <property name="rightMargin">
         <number>50</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="topMargin">
           <number>50</number>
          </property>
          <property name="bottomMargin">
           <number>50</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="allCostLabel">
              <property name="text">
               <string>Общая стоимость:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="allCostButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius:10px;
background-color:#007700;

</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="remainingLabel">
              <property name="text">
               <string>Осталось:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="remainingButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius:10px;
background-color:#007700;
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="paidLabel">
              <property name="text">
               <string>Оплачено:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="paidButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius:10px;
background-color:#007700;
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="monthlyLabel">
              <property name="text">
               <string>Оплата за месяц:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="monthlyButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius:10px;
background-color:#007700;
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="paidMonthsLabel">
              <property name="text">
               <string>Оплаченные месяцы:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="paidMonthsButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius:10px;
background-color:#007700;
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="missedMonthsLabel">
              <property name="text">
               <string>Пропущенные месяцы:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="missedMonthsButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius:10px;
background-color:#007700;
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="arrearsLabel">
              <property name="text">
               <string>Должен:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="arrearsButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius:10px;
background-color:#007700;
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="currencyLabel">
              <property name="text">
               <string>Валюта:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="currencyButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius:10px;
background-color:#007700;
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableView" name="tableView">
            <property name="minimumSize">
             <size>
              <width>500</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>1200</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="bottomTable">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>70</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>698</number>
     </property>
     <property name="rightMargin">
      <number>45</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ToEXCELButton">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>52</height>
        </size>
       </property>
       <property name="text">
        <string>ЭКСПОРТ В EXCEL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ToPDFButton">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>52</height>
        </size>
       </property>
       <property name="text">
        <string>ЭКСПОРТ  В PDF</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
